docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -0,0 +1,54 @@
+Referencing Content Blocks in Library V2
+--------------------------------------------------
+
+Status
+=======
+Pending
+
+Context
+=======
+Currently, the courses are being stored in the Modulestore and we want the library content to be stored in the Blockstore.
+To achieve this we need a referencing mechanism to reference the libraries to the courses. This referencing helps in replacing
+v1 libraries (current content libraries stored on Modulestore) with v2 libraries (Blockstore backed) which are:
+
+#. not dependent on edx-platform
+#.simpler to maintain than modulestore
+#. increases re-usability of the content
+
+Terminology Used
+^^^^^^^^^^^^^^^^
+#. Blocks: Each reusable piece of content in a library. Example: problems, videos, HTML
+#. Library: Collection of blocks in a course.
+
+
+Decisions
+=========
+We want to be able to reference single and multiple specific blocks from a library (v2, as well as v1) into a course (that are stored in modulestore)
+The workflow discussed for the interface for library content referencing would be:
```

Comment:
Can you please explain how this is different from the existing "[Library Sources Content Block](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/library_sourced_block.py)" ?

Reply:
@bradenmacdonald doesn't the implementation section highlight the difference?

Reply:
If so, I don't see it? That section explains how this new block would work, but it doesn't explain how the existing one works, or compare them.